third_party/move/move-compiler/src/expansion/translate.rs

```diff
@@ -573,6 +587,25 @@ fn script_(context: &mut Context, package_name: Option<Symbol>, pscript: P::Scri
     }
 }
 
+// If attributes contains a #![deprecated] attribute, then returns the location of the attribute.
```

Comment:
Why `#!` instead of `#`? Also please use `///` for documentation comments, and inside the comment backslashes.

Reply:
Sorry, typo.  Too many different languages use #![ in different ways.